integration_test/setup.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
This is (almost?) the same script as in the computation engine.
Would there be a way to use that one?

Reply:
I removed the stuff from influxdb as I removed influxdb also from this setup. However, your comment speaks to a greater concern that I have. This integration test uses the same infrastructure of computation-engine except it puts some stuff in development mode. I would like to have this test available long before the orchestrator is released and I would like to also be able to use development versions of other components in these types of integration tests. In other words, putting this test in the computation-engine repo might be a bad fit.

However, in order to use 1 component properly, you need the other components so regardless of the integration test, they all depend on the complete infrastructure. Creating mock components just for testing purposes, I think, is a bridge too far for this project.

I am not happy about the placement either. Perhaps we should start investigating if we would rather create a development-mode in computation-engine that allows locally-run (outside of docker) development versions of components. A requirement would be that we can run single-component integration tests inside the CI pipeline of that component (so we don't have to release it in order to test it). Then we create it in such a way that we can pick and choose which components become available. In order to not duplicate the computation-engine infrastructure everywhere, perhaps a Git submodule might be the key. 

Might be worthwhile to evaluate if the postgres config for the orchestrator is also in the right spot. Perhaps we should move that to the orchestrator repo and also use a git submodule to reference the same postgresql database. Normally you would setup a postgresql database per component and not bother with centralizing it, but that would complicate the docker-compose.yml in computation-engine.

All-in-all, some stuff to discuss. We need a refactor now that everything is becoming more mature. To close off this comment, I propose we keep it like this for now, so we can clean it up with the cleanup I was describing in the previous paragraphs.